### ISSUE TITLE
Upgrading from previous versions

### ISSUE THREAD

**Issue by DirtyF** [netlify/cli#106]
Hi,

Just want to let you know that I encountered some error message after installing next version of the package.

Apparently `siteId` is now stored in `.netlify/state.json` but on first run on an existing project CLI commands will throw an error:

```sh
Error: ENOTDIR: not a directory, open '/Users/frank/code/frank.taillandier.me/.netlify/state.json'
    at StateConfig.get all [as all] (/usr/local/lib/node_modules/netlify-cli/src/base/state/index.js:18:28)
    at StateConfig.get (/usr/local/lib/node_modules/netlify-cli/src/base/state/index.js:64:29)
    at Object.get (/usr/local/lib/node_modules/netlify-cli/src/base/site-config/index.js:17:22)
    at InitCommand.run (/usr/local/lib/node_modules/netlify-cli/src/commands/init.js:23:25)
```

I had to delete the current `.netlify` directory and run `netlify init` to get going.

I would expect at least some instructions on how to migrate for existing users.

Thanks and keep up the good work 💪

**Comment by bcomnes** [comment]
Agreed this is definitely not the migration path we intended.

**Comment by DirtyF** [comment]
Maybe `netlify` command could check for existing `.netlify` and copy existing `siteId` to `.netlify/state.json` if it finds one?

**Comment by bcomnes** [comment]
Should be fixed in `2.0.0-beta.3`

`npm i -g netlify-cli@next`
